src/fontra/client/css/plugins.css

```diff
@@ -0,0 +1,9 @@
+.plugins-page {
+  padding: 2rem;
+  font-size: 1.1rem;
```

Comment:
This has the effect of the Dialog _also_ being 10% larger than normal. I think the scope of the font size should be smaller.